packages/react-frontend/src/pages/QuestionCatalog.js

```diff
@@ -16,7 +16,7 @@ function QuestionCatalog() {
 
   return (
     <div className="Frontpage">
-      <Navbar navbar>navbar</Navbar>
+      <Navbar navbar setQuestions={ setQuestions }>navbar</Navbar>
```

Comment:
I worry about what might happen if the search bar is used on other pages. This is fine for now, but in the future we might want to change this to avoid calling the function which doesn't exist.